supplier_api_tester/supplier_api_tester/v1/tests/booking.py

```diff
@@ -239,3 +241,39 @@ def test_cancellation(api_url, api_key, product_id, timeslots: bool, version=1):
     check_api_error(raw_response, api_error, expected_error)
     return TestResult()
 
+
+@test_wrapper
+def test_barcodes(api_url, api_key, product_id, timeslots: bool, version=1):
```

Comment:
In case of API testing the approach is different from the pytest when we have multiple unit tests and each test is testing one thing only. Here we're trying to make as few API calls as possible and make all kind of tests based on a single response so it would be better to extend the existing test then adding another one.